timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -448,13 +453,43 @@ export class TimeGraphChart extends TimeGraphChartLayer {
         this.isNavigating = flag;
     }
 
+    setEventClickedFlag(flag: boolean) {
+        this.isEventClicked = flag;
+    }
+
+    private selectStateOnEventClicked() {
+        const selectedRow = this.rows.find(row => {
+            if (row && this.unitController.selectionRange) {
+                const cursorPosition = this.unitController.selectionRange.end;
+                const rowElement = row.states.find((rowElementModel: TimelineChart.TimeGraphState) => rowElementModel.range.start === cursorPosition
```

Comment:
I think we need to find a better way to sync the row than relying the on the timestamp. With nanosecond precision and a huge number of events it will be click on the trace event of start or end state.
I think the row to select needs to determined outside the library and the library needs to provide an API to show (reveal) the selected row.